.changeset/modern-ghosts-sin.md

```diff
@@ -0,0 +1,31 @@
+---
+"@logto/tunnel": minor
+---
+
+add deploy command and env support.
+
+#### Add new `deploy` command to deploy your local custom UI assets to your Logto Cloud tenant
+
+1. Create a Machine-to-Machine app in your Logto tenant
+2. Run the following command
+
+```bash
+npx @logto/tunnel deploy --app-id <your-m2m-app-id> --app-secret <your-m2m-app-secret> --endpoint https://<tenant-id>.logto.app --experience-path /path/to/your/custom/ui
```

Comment:
does it work for custom domains? if no, shall we remove or improve the success message?

Reply:
Good point. Will test against custom domain and update accordingly